modele/app.model.js

```diff
@@ -8,8 +8,10 @@ exports.gettingTopics = () => {
   });
 };
 exports.gettingArticles = (article_id) => {
-  const query = "SELECT * FROM articles WHERE article_id = $1;";
+  if (article_id) {const query = "SELECT * FROM articles WHERE article_id = $1;";
   return db.query(query, [article_id]).then((result) => {
     return result.rows[0];
   });
+}
+return Promise.reject({ status: 404, msg: 'custom error'})
```

Comment:
this could be a better error message, just bad request would be fine, but it would be good to use it